src/test/java/com/google/api/codegen/testdata/go/go_library.baseline

```diff
@@ -36,7 +36,7 @@ var _ = iterator.Done
 var _ = strconv.FormatUint
 var _ = time.Now
 
-func TestLibraryServiceSmoke(t *testing.T) {
+func TestLibClientSmoke(t *testing.T) {
```

Comment:
I think we are missing `Lib` in a couple of places.
1. The path template functions should be updated too. `ShelfPath` should become `ShelfLibPath`.
2. The file name should probably become `lib_client.go` etc. This one isn't strictly required.